packages/core-components/src/components/TabbedLayout/RoutedTabs.tsx

```diff
@@ -68,35 +64,33 @@ export function useSelectedSubRoute(subRoutes: SubRoute[]): {
 
 export function RoutedTabs(props: { routes: SubRoute[] }) {
   const { routes } = props;
-  const navigate = useNavigate();
+
   const { index, route, element } = useSelectedSubRoute(routes);
   const headerTabs = useMemo(
     () =>
-      routes.map(t => ({
-        id: t.path,
-        label: t.title,
-        tabProps: t.tabProps,
-      })),
+      routes.map(t => {
+        const { path, title, tabProps } = t;
+        let to = path;
+        // Remove trailing /*
+        to = to.replace(/\/\*$/, '');
+        // And remove leading / for relative navigation
+        to = to.replace(/^\//, '');
+        return {
+          id: path,
+          label: title,
+          tabProps: {
```

Comment:
Is it possible to spread tabProps at the end, so that the value for `to`/`component` can be overridden?

```
tabProps: {
            component: Link,
            to,
            ...tabProps,
},
```